### ISSUE TITLE
Add `/*` suffix to aliases to make them work

### ISSUE THREAD

**Issue by FunctionDJ** [Subwaytime/vite-aliases#28]
Out of the box, `vite-aliases` doesn't add `/*` suffixes to path *values*, which from my experience is required to make them work.
(Maybe just newer versions?)

`vite-aliases`:
```json
"@interfaces/*": ["C:/.../interfaces"]
```
Working for me:
```json
"@interfaces/*": ["C:/.../interfaces/*"]
```

Maybe a `suffix` config option to make this work?

**Comment by Subwaytime** [comment]
Just realized that i already fixed this issue, but its not added to the Path only to the key.. , thats on my end! 😅

**Comment by Subwaytime** [comment]
fixed with [v0.8.4](https://github.com/Subwaytime/vite-aliases/releases/tag/v0.8.4)
